content/api/servo-assembly/servos.md

```diff
@@ -2,11 +2,11 @@
 title: Servos
 ---
 
-From the servo assembly, you can control the attached servos. By default, servos will be unpowered when your robot starts, and can freely rotate. Once you set a value, they then fix to that rotation. This also happens when you turn off your robot, or unplug your USB stick.
+You can control attached servo motors from the servo assembly. By default, servos will be unpowered when your robot starts, and can freely rotate when turned by hand. Upon setting a value, they will hold the corresponding position. They will become unpowered again when you turn off your robot, or unplug your USB stick.
```

Comment:
they also go unpowered when the position is set to `None`. Not sure if that needs to be mentioned here, or if it's ok to just be mentioned below?

Reply:
Might as well add it here too.